.github/workflows/ci.yml

```diff
@@ -1,9 +1,7 @@
 name: Tests Pipeline
 
 on:
-  pull_request:
-  push:
-    branches: ['main']
+  pull_request: {}
```

Comment:
Why this change? GitHub is not helpful in pointing me to the commit that introduced it.